### DISCUSSION TITLE
Plugin's default priority adjust

### DISCUSSION THREAD

**Discussion by onesounds** [Flow-Launcher/Flow.Launcher#733] [Ideas]
**the current situation**
- All plugin's default priority is 0.

**Issue Case**
- in case default a: 0 :b 0 :c 0, when I wanna change b priority to down, I have to change a & c to 1.

**How To Solve**
- All plugin's priority default to 1 or 2.

**Additional**
- It would be better to give a more optimized default setting. like this.
- System Control : 3  <- Very important
- Web Search : 0 <- in case, When I using search for just press up arrow..
- Program 1 / Everything 1 / Explorer 1 <- I don't know if this is right. It depends on what usually use.

In summary, Control >Program Search things>Web search... 
It's good to set it differently and in detail for each person, but I think it would be good if it was set in this way when it was first used.

**Comment by jjw24** [comment]
>All plugin's priority default to 1 or 2.

i think setting it to 1 or 2 as default is fine. 

>In summary, Control >Program Search things>Web search...

I would think it's better to have all plugin equal and let users decide and adjust, because everyone have their own order. for me as an example i wouldnt even use system control that much.
